rskj-core/src/test/java/org/ethereum/vm/ProgramTest.java

```diff
@@ -34,7 +34,7 @@
 import static org.mockito.Mockito.*;
 
 
-public abstract class ProgramTest {
+public class ProgramTest {
```

Comment:
why removing the abstract here?

Reply:
I initially struggled  trying to run just this test. The way it was implemented, the `ProgramTest` could not be executed. It would allow you to execute one of the inheritors. And this abstraction created just to override the beforeEach seemed overkill to me. So I removed a test file and added one with logs and gas enabled by default and made the others override the `beforeEach` method.  It made more sense than have a test class with test code that can't be executed alone and rely others to initialise their objects. 🤷‍♂️ 

For me, it made more sense. This way if we want to add a test, we edit only the `ProgramTest` class and the others will execute this test since they inherit and override the method with the flags.

Reply:
got it, that makes sense. Thanks for clarifying!